<ui version="4.0" >
 <class>accountseditfrm</class>
 <widget class="QDialog" name="accountseditfrm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit entry...</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/images/first4.qrc" >:/kcalc.png</iconset>
  </property>
  <widget class="QLabel" name="lbl_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>94</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Reference Nr.:</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRefNr" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>50</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_6" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>380</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Amount:</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbladdrID" >
   <property name="geometry" >
    <rect>
     <x>370</x>
     <y>210</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>-</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Entry-ID:</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_8" >
   <property name="geometry" >
    <rect>
     <x>379</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Type:</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_7" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>120</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Address:</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtamount" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>380</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_3" >
   <property name="geometry" >
    <rect>
     <x>381</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Date:</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>90</y>
     <width>38</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Code:</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbincexp" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>90</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <item>
    <property name="text" >
     <string>Income</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Expenditure</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbl_4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>260</y>
     <width>92</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Description:</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCode" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>90</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblID" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>-</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="date1" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>50</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="displayFormat" >
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btncancel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>430</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
   <property name="icon" >
    <iconset resource="../resources/images/first4.qrc" >:/button_cancel.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnaccept" >
   <property name="geometry" >
    <rect>
     <x>450</x>
     <y>430</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Accept</string>
   </property>
   <property name="icon" >
    <iconset resource="../resources/images/first4.qrc" >:/button_ok.png</iconset>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtdescription" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>260</y>
     <width>451</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtaddress" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>120</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnsearch" >
   <property name="geometry" >
    <rect>
     <x>370</x>
     <y>120</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="icon" >
    <iconset resource="../resources/images/first4.qrc" >:/xmag.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnclearsearch" >
   <property name="geometry" >
    <rect>
     <x>370</x>
     <y>150</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="icon" >
    <iconset resource="../resources/images/first4.qrc" >:/button_cancel.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btncodecheck" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>90</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="icon" >
    <iconset resource="../resources/images/first4.qrc" >:/button_ok.png</iconset>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtRefNr</tabstop>
  <tabstop>date1</tabstop>
  <tabstop>txtCode</tabstop>
  <tabstop>btncodecheck</tabstop>
  <tabstop>cmbincexp</tabstop>
  <tabstop>txtaddress</tabstop>
  <tabstop>btnsearch</tabstop>
  <tabstop>btnclearsearch</tabstop>
  <tabstop>txtdescription</tabstop>
  <tabstop>txtamount</tabstop>
  <tabstop>btnaccept</tabstop>
  <tabstop>btncancel</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/images/first4.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>btnaccept</sender>
   <signal>clicked()</signal>
   <receiver>accountseditfrm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btncancel</sender>
   <signal>clicked()</signal>
   <receiver>accountseditfrm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
